dockertest_test.go

```diff
@@ -76,7 +77,7 @@ func TestMongo(t *testing.T) {
 		}
 
 		if response.StatusCode != http.StatusOK {
-			return fmt.Errorf("Could not connect to resource.")
+			return fmt.Errorf("could not connect to resource")
```

Comment:
While lower case starting characters for sentences are actually the way to go in Go, adding a dot at the end of the string is not :) But it doesn't really matter here, just dropping my 2 cents :)